src/main/java/lottery/domain/ShufflePicker.java

```diff
@@ -0,0 +1,21 @@
+package lottery.domain;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.IntStream;
+
+public class ShufflePicker implements Picker {
+    private final List<Integer> basis;
+
+    public ShufflePicker(int min, int max) {
+        basis = IntStream.rangeClosed(min, max).collect(ArrayList::new, List::add, List::addAll);
+    }
+
+    @Override
+    public List<Integer> pick() {
+        List<Integer> candidate = new ArrayList<>(basis);
+        Collections.shuffle(candidate);
```

Comment:
```suggestion
        Collections.shuffle(basis);
```
굳이 새로 만들지 않아도 될 것 같아요!

Reply:
수정: 38a5ee2